build.sbt

```diff
@@ -71,6 +73,7 @@ libraryDependencies ++= {
   val scalaLoggingVersion = "3.9.4"
   val snappyVersion = "1.1.8.4" // Support Apple Silicon
   val scalatestVersion = "3.2.0"
+  val hadoopVersion = "3.1.4"
```

Comment:
i thought we were on 3.2.x at this point?

Reply:
OH, my bad.  Will update it to 3.2 then